frameworks/kafka/README.md

```diff
@@ -1,14 +1,350 @@
-# DC/OS Apache Kafka Service Documentation
-
-## Table of Contents
-
-- [Overview](docs/index.md)
-- [Quick Start](docs/quick-start.md)
-- [Install and Customize](docs/install-and-customize.md)
-- [Uninstall](docs/uninstall.md)
-- [Configure](docs/configure.md)
-- [Connecting Clients](docs/connecting-clients.md)
-- [Managing](docs/managing.md)
-- [API Reference](docs/api-reference.md)
-- [Troubleshooting](docs/troubleshooting.md)
-- [Limitations](docs/limitations.md)
+<p align="left"><img src="https://mesosphere.com/wp-content/themes/mesosphere/library/images/assets/dcos-sdk-logo.png" width="250"/></p>
+
+
+# Apache Kafka Service Guide
+
+<!-- Add TOC later if needed, once all sections are finalized
```

Comment:
This comment wasn't terminated, making the whole doc a comment. I terminated in my edit. Though I'll need someone to read through to make sure I haven't uncommented things that should be commented.